### ISSUE TITLE
Is there something wrong about data sets in your paper?

### ISSUE THREAD

**Issue by RunZhengPeng** [johanna-rock/imRICnn#1]
In your paper , you said there are 2000 data sets for training . While  i download the data sets , I find that it's size is 1024*25600 . If 1024*128 act as a RD map as you described , and then there are only 200 scenarios . Hope to get advice .
    my email : pzheng0822@gmail.com .

**Comment by johanna-rock** [comment]
That's right. For the results in the paper, I used the whole data set of 2000 (training) + 250 (validation) + 250 (test) examples (= RD maps). The data set from the download is only a subset with 200 (training) + 25 (validation) + 25 (test) examples. The reason for this is its large size. The subset is sufficient for testing the pre-trained models and for demonstration purposes. If you are interested in the whole data set for more extensive testing and training, I could temporarily make it available for download.
Note: Training data examples contain only the signal from one antenna, while validation and testing examples contain the signal from eight antennas (for validation measures concerning the angle of arrival).

**Comment by RunZhengPeng** [comment]
Thank you so much.  I want the whole data set for more research. Where can i download it ?

**Comment by RunZhengPeng** [comment]
I treat the data set as 1024 samples * 128 chirps * 200 frames . I take out the first chirp of frame 3(contain 2 targets) and plot in time and frequency domain .  data called "s_IF" and "s_IF_zero_interf_td"  look the same . In fact , I want the data that include with interference and without interference . But i don't know the meaning of different data . So could you explain for this ? Thank you .

**Comment by johanna-rock** [comment]
What you are looking for are the data contained in s_IF (with interference) and s_IF_clean (without interference).
s_IF: IF signal from object reflections + AWGN + interference
s_IF_clean: IF signal from object reflections

Also contained in the data:
s_IF_clean_noise: signal from object reflections + AWGN
s_IF_zero_interf_td: signal from object reflections + AWGN where all time domain samples that are affected by interference are set to zero.
objects: contains the true ranges velocities and angles of the simulated point objects.

Note: The data is already in frequency domain (after the first FFT over the fast time samples within one ramp).

**Comment by RunZhengPeng** [comment]
Thanks so much. pzheng0822 邮箱pzheng0822@gmail.com 签名由 网易邮箱大师 定制 On 08/21/2019 17:05, Johanna Rock wrote: What you are looking for are the data contained in s_IF (with interference) and s_IF_clean (without interference). s_IF: IF signal from object reflections + AWGN + interference s_IF_clean: IF signal from object reflections Also contained in the data: s_IF_clean_noise: signal from object reflections + AWGN s_IF_zero_interf_td: signal from object reflections + AWGN where all time domain samples that are affected by interference are set to zero. objects: contains the true ranges velocities and angles of the simulated point objects. Note: The data is already in frequency domain (after the first FFT over the fast time samples within one ramp). — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.
